src/components/DotNavigation/DotNavigation.visual.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TPAComponentsProvider } from '../TPAComponentsConfig';
+import { VisualContainerElement } from '../../../test/visual/VisualContainerElement';
+import { Theme, DotNavigation } from './DotNavigation';
+
+class DotNavigationVisual extends React.Component<any> {
+  static defaultProps = {
+    mobile: false,
+  };
+
+  render() {
+    const { mobile } = this.props;
+
+    return (
+      <TPAComponentsProvider value={{ mobile }}>
```

Comment:
We can remove all references to the `TPAComponentsProvider` and `TPAComponentsConsumer` and the `mobile` state, because I think we don't have any special style for it

Reply:
Done)